pkg/archive/archive_windows.go

```diff
@@ -26,15 +26,8 @@ func getWalkRoot(srcPath string, include string) string {
 // CanonicalTarNameForPath returns platform-specific filepath
 // to canonical posix-style path for tar archival. p is relative
 // path.
-func CanonicalTarNameForPath(p string) (string, error) {
-	// windows: convert windows style relative path with backslashes
-	// into forward slashes. Since windows does not allow '/' or '\'
-	// in file names, it is mostly safe to replace however we must
-	// check just in case
-	if strings.Contains(p, "/") {
-		return "", fmt.Errorf("Windows path contains forward slash: %s", p)
-	}
-	return strings.Replace(p, string(os.PathSeparator), "/", -1), nil
+func CanonicalTarNameForPath(p string) string {
```

Comment:
That seems same as `filepath.ToSlash()`

Reply:
Ha, yes, indeed. Not sure how long it's been like that. Will update.

Reply:
Pushed update

Reply:
I thought just removing this function completely but that can be a follow-up.